usaspending_api/submissions/management/commands/reveal_dabs_submission_window_schedule.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import pytz
+
+from datetime import datetime
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from usaspending_api.submissions.models import DABSSubmissionWindowSchedule
+
+logger = logging.getLogger("script")
+
+
+class Command(BaseCommand):
+    """
+    This command updates the 'submission_reveal_date' field on dabs_submission_window_schedule
+    table entries that are ready to be revealed. When the 'submission_due_date' for monthly
+    schedules or the 'certification_deadline' for quarterly schedules is reached, the
+    'submission_reveal_date' is set to the current time. This command is intended to be run
+    after submissions finish loading in the nightly pipeline.
+    """
+
+    help = "Updates Reveal Dates in the DABS Submission Window Schedule table."
+
+    @transaction.atomic()
+    def handle(self, *args, **options):
+
+        logger.info("Loading existing DABS Submission Window Schedules")
+        existing_schedules = DABSSubmissionWindowSchedule.objects.all()
+
+        now = datetime.now(tz=pytz.UTC)
+
+        for schedule in existing_schedules:
+
+            if schedule.is_quarter:
```

Comment:
Is this check necessary? Appears that `submission_due_date` works for both.

Reply:
This has been removed